### ISSUE TITLE
Order of point is incorrect

### ISSUE THREAD

**Issue by tfzxyinhao** [codeniko/shape_predictor_81_face_landmarks#7]
Order of additional 13 points is incorrect

**from 69 to 81**

![image](https://user-images.githubusercontent.com/444814/54862312-c7318580-4d73-11e9-98f7-88d6ff641175.png)

**Comment by codeniko** [comment]
Yes, you're right. This was something I built without too much thought in the order and it didn't bother me for my project. I was more interested to make sure the points were good and positioning was symmetrical.  I need to find some time to fix this but I don't have it planned at the moment
